front/src/styles/scss/applications/stdcmV2/_card.scss

```diff
@@ -61,39 +61,101 @@
   }
 
   .stdcm-v2-card__body {
-    padding: 0.688rem 1rem 0.8125rem 1.5rem;
-    // TODO: remove this style after ui-core component is implemented
-    .suggestions * label,
-    .stdcm-v2-via-stop-for * label {
+    padding: 14px 24px 16px 8px;
+
+    .location-line {
+      padding-left: 16px;
```

Comment:
```suggestion
```

Reply:
I think I need to keep this padding-left, otherwise I have this:
![image](https://github.com/user-attachments/assets/254cbd58-3a3d-4a6b-94c8-5ec0d30dae65)